### DISCUSSION TITLE
How to use rounding in Invoice/Offer

### DISCUSSION THREAD

**Discussion by mad99** [YetiForceCompany/YetiForceCRM#14389] [How to]
Hi,
Can't find a solution in documentation/Git-Discussions.

In Switzerland the minimal coin-value is 5 Rp. (Five «Rappen»). So any bill, invoice or offer is automatically mathematical roundet on 5Rp.

So «CHF 97.77» would automatic roundedt to «CHF 97.75».
Is there some build-in solution to round invoice/offer on 5 "units" depending on the currency?

Currently I have to List a invoice/offer Item «Rundungsdifferenz» (Rounding difference) and then do it manually -0.02 in the example above.
Not really nice looking and absolutely unusual in Switzerland. Typically the Item is roundet at its own.
Would be great to get some hint.

- Maybe there is some settings for, or some other built-in solution?

**Comment by vovpff** [comment]
I think there is no tools for this available in system.

You can implement rounding manualy by changing core files.

**Comment by mariuszkrzaczkowski** [comment]
there is no such mechanism

**Reply by mad99** [reply]
OK
Thank you @vovpff and @mariuszkrzaczkowski.
So I don't have to search anymore ;)
Will find an other solution.

**Reply by mariuszkrzaczkowski** [reply]
it can only be done requires writing it
if you are interested, write to us, we will provide a quote
